test/no-array-instanceof.js

```diff
@@ -31,38 +31,23 @@ ruleTester.run('avoid-array-instanceof', rule, {
 			errors
 		},
 		{
-			code: 'arr instanceof Array === true',
-			output: 'Array.isArray(arr) === true',
-			errors
-		},
-		{
 			code: '[] instanceof Array',
 			output: 'Array.isArray([])',
 			errors
 		},
 		{
-			code: '[] instanceof Array === true',
-			output: 'Array.isArray([]) === true',
-			errors
-		},
-		{
-			code: '[1,2,3] instanceof Array',
-			output: 'Array.isArray([1,2,3])',
-			errors
-		},
-		{
-			code: '[1,2,3] instanceof Array === true',
-			output: 'Array.isArray([1,2,3]) === true',
+			code: 'fun.call(1, 2, 3) instanceof Array',
+			output: 'Array.isArray(fun.call(1, 2, 3))',
 			errors
 		},
 		{
-			code: 'if(arr instanceof Array){}',
-			output: 'if(Array.isArray(arr)){}',
+			code: 'obj.arr instanceof Array',
+			output: 'Array.isArray(obj.arr)',
 			errors
 		},
 		{
-			code: 'if([1,2,3] instanceof Array){}',
-			output: 'if(Array.isArray([1,2,3])){}',
+			code: `foo.bar${value} instanceof Array`,
```

Comment:
Same as before. Do you mean to have a test with a template literal? Because as it is right now, you are making a test case that will evaluate to `foo.bar2 instanceof Array`